samples/cpp/stitching_detailed.cpp

```diff
@@ -120,7 +120,9 @@ static void printUsage()
         "  --output <result_img>\n"
         "      The default is 'result.jpg'.\n"
         "  --timelapse (as_is|crop) (range_width)\n"
-        "      Output warped images separately as frames of a time lapse movie, with 'fixed_' prepended to input file names.\n";
+        "      Output warped images separately as frames of a time lapse movie, with 'fixed_' appended before extension input file names.\n"
+        "  --rangewidth <int>\n"
+        "      uses range_width to limit number of images to match with.\n";
```

Comment:
This changes makes sense because OpenCV CommandLineParser can process only one parameter value at once (There is somewhere a PR with samples migration to CommandLineParser).

- `--timelapse (as_is|crop) (range_width)`: need to fix message and remove " (range_width)"
- my suggestion is to rename rangewidth to timelapse_range/timelapse_rangewidth (because it is used only with timelapse)
- `fixed_`/`_fixed` message (see the separate note below)